### PR TITLE
[9.x] Allow underline in numeric validation values

### PR DESCRIPTION
**Pull request by imanghafoori1**
PHP 7.4 allows putting the underline character in integers for more readability, like:
"1_500_000" instead of: "1500000"
This PR allows putting underline in numbers of validation rules: ```Numeric|Between:500_000,1_000_000```

- It also adds some important missing tests.

### CODE REVIEW

File: src/Illuminate/Validation/Concerns/ValidatesAttributes.php [hunk 1]
```diff
@@ -407,7 +407,7 @@ public function validateBetween($attribute, $value, $parameters)
 
         $size = $this->getSize($attribute, $value);
 
-        return $size >= $parameters[0] && $size <= $parameters[1];
+        return $size >= (int) str_replace('_', '', (string) $parameters[0]) && $size <= (int) str_replace('_', '', (string) $parameters[1]);
```
Comment (lucasmichot) [inline src/Illuminate/Validation/Concerns/ValidatesAttributes.php:410]: They should be casted to float, as the boundaries might not be integers.

Comment (imanghafoori1) [reply src/Illuminate/Validation/Concerns/ValidatesAttributes.php:410]: Is it better to remove the (int) cast and leave it to PHP?

Comment (lucasmichot) [reply src/Illuminate/Validation/Concerns/ValidatesAttributes.php]: I would add extra tests to cover this case and adjust your code accordingly

Comment (imanghafoori1) [reply src/Illuminate/Validation/Concerns/ValidatesAttributes.php]: I force pushed some more tests to cover the floats.

**Pull request closed without merging**